Dockerfile

```diff
@@ -0,0 +1,14 @@
+FROM oven/bun:latest
+
+WORKDIR /app
+
+COPY .env ./
```

Comment:
`.env` is for local and should not be pushed to remote. Make it difficult building as github actions. Rethink